### PR TITLE
Gmail reply mail

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29245

## Description
- Added a new command called `reply-mail`
- Fixed an issue where additional headers were added only if attachments were received.
- Fixed an issue where attachments were not attached correctly when both `htmlBody` and `body` arguments were received.
- Updated the Docker image to: *demisto/google-api:1.0.0.13485*
- Updated README with new command and converted it to md

#### Note
This has been tested on https://ec2-54-220-29-9.eu-west-1.compute.amazonaws.com

### CODE REVIEW

File: Packs/Gmail/Integrations/Gmail/README.md [hunk 1]
```diff
@@ -6,56 +6,39 @@ This integration replaces the Gmail functionality in the GoogleApps API and G Su
 
 There are several procedures you have to perform in Google before configuring the integration on Demisto.
 
-* [Get a New Private Key](#get-a-new-private-key)
-* [Delegate Domain-wide Authority to Your Service Account](#delegate-domain-wide-authority-to-your-service-account)
-* [Get an Immutable Google Apps ID](#get-an-immutable-google-apps-id-parameters)
+* Get a New Private Key
+* Delegate Domain-wide Authority to Your Service Account
+* Get an Immutable Google Apps ID
```
Comment (yaakovi) [inline Packs/Gmail/Integrations/Gmail/README.md:11]: why did you remove the links?

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 2]
```diff
@@ -1735,6 +1735,12 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
     message['subject'] = header(subject)
     message['reply-to'] = header(replyTo)
 
+    # The following headers are being used for the reply-mail command.
+    if inReplyTo:
+        message['In-Reply-To'] = header(' '.join(inReplyTo))
+    if references:
```
Comment (yaakovi) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:1741]: what does `references` do?

File: Packs/Gmail/Integrations/Gmail/README.md (hunk 1 above)
Comment (Bargenish) [reply Packs/Gmail/Integrations/Gmail/README.md:11]: missed that while merging the conflict. i'll revert it.

File: Packs/Gmail/Integrations/Gmail/Gmail.py (hunk 2 above)
Comment (Bargenish) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:1741]: Similar to `In-Reply-To`, you refer to the message you're replying to but this header can also contain more message ids in order to create a "threaded conversation" and see all the messages that are linked to this one so one could find it on his mailbox.
https://tools.ietf.org/html/rfc2822#appendix-A.2

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 3]
```diff
@@ -1992,6 +2041,7 @@ def main():
         'gmail-delegate-user-mailbox': delegate_user_mailbox_command,
         'gmail-remove-delegated-mailbox': remove_delegate_user_mailbox_command,
         'send-mail': send_mail_command,
+        'reply-mail': reply_mail_command,
```
Comment (yaakovi) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:2044]: update the detailed description file with the additional command.

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.  Reviewed the Release Notes and the new ***reply-mail*** command that was added to the yml and README.
